tests/benchdnn/graph/deserialize.cpp

```diff
@@ -516,6 +524,83 @@ const deserialized_op_t &deserialized_graph_t::get_op_by_in_lt(
     return dummy;
 }
 
+void deserialized_graph_t::detect_recognized_patterns() {
```

Comment:
Probably needs a note that it's intended to be called once from the specific spot and, thus, doesn't have extra handling of the fact of single calling intention like extra state or logic.